openaerostruct/aerodynamics/pg_scale.py

```diff
@@ -205,17 +205,17 @@ def compute_partials(self, inputs, partials):
         rotational = self.options["rotational"]
 
         M = inputs["Mach_number"]
-        betaPG = np.sqrt(1 - M ** 2)
-        fact = np.array([1.0, betaPG, betaPG], M.dtype).flatten()
-        fact_norm = np.array([betaPG, 1.0, 1.0], M.dtype).flatten()
+        betaPG = np.sqrt(1 - M ** 2)[0]

```

Comment:
All OM inputs are cast to ndarray right? I don't like the `[0]`, it's probably better to use `np.asscalar()`

Reply:
I didn't know `asscalar` existed, I agree that's a better looking solution